# replace this with boat ip address 
IP: "127.0.0.1"
PORT: 8080
# IP: "192.168.0.50"
# PORT: 8001


# range of angular velocity 
V_MAX: 100.0
V_MIN: 0.0
# range of angular velocity 
W_MAX: 30.0
W_MIN: -30.0
# sr vessel template msg 
cmd_str_msg: "$PSEAC,T,,{},{},*"
